package.json

```diff
@@ -29,6 +30,12 @@
     "simd": "^2.0.0"
   },
   "devDependencies": {
+    "babel-cli": "^6.5.1",
```

Comment:
I'm really not sure `babel-cli` is needed.

AFAICT, the `babel` executable is installed as part of `babel-core`, and `babel-cli` is meant to be installed globally (`npm install -g babel-cli`) so it's available on your command-line anywhere, and will "sniff" the current module to actually call the locally-installed `babel` (at `node_modules/.bin/babel`) if it exists; defaulting to the globally-installed one otherwise.